composer.json

```diff
@@ -5,11 +5,11 @@
     "homepage": "https://github.com/DASPRiD/container-interop-doctrine",
     "license" : "BSD-2-Clause",
     "require": {
-        "php": "^5.5|^7.0",
+        "php": "^5.5|^5.6|^7.0|^7.1",
```

Comment:
strictly speaking, because PHP is not semver, compatible, we should be more specific about version requirements here; i.e. something like `5.5.*|5.6.*|7.0.*|7.1.*` - what do you think?